packages/nitro-protocol/src/signatures.ts

```diff
@@ -48,6 +52,21 @@ export function signState(state: State, privateKey: string): SignedState {
   return {state, signature};
 }
 
+export async function fastSignState(
+  state: State,
+  privateKey: string
+): Promise<{state: State; signature: string}> {
+  const wallet = new Wallet(privateKey);
```

Comment:
It is also worth investigating how slow this operation is: constructing a `Wallet` will use an `'elliptic'` elliptic curve to get the `address`.

We could use the equivalent function from `libauth` instead.